src/compare-pdf-to-snapshot.ts

```diff
@@ -72,6 +73,11 @@ export const comparePdfToSnapshot = (
   const snapshotPath = join(dir, snapshotName + '.png')
 
   if (!existsSync(snapshotPath)) {
+    if (mode === 'failFast') {
+      throw new Error(
+        `${snapshotPath} does not exist. Run comparePdfToSnapshot in create mode to create it.`,
+      )
+    }
```

Comment:
I would prefer to return a boolean value. Do you really want everything to halt in this mode? If it is the later, maybe an option would be to add some sorts of hooks to the options. One hook could be called here and then you'd be able to simply throw in it, which should get you the same result?
WDYT?